brainstormforce/ultimate-addons-for-gutenberg
# Fix: Image - Resolve an issue of image caption visible twice in editor.

### Description

- Image unwanted bottom space for caption

### Screenshots

- https://share.getcloudapp.com/2Nubj1WK

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Switch to `next-release` or master
- Activate the Gutenberg plugin
- Drag-drop the image block
- Select the image and check the unwanted bottom space.
- Now enable caption, and add some caption.
- You will see 2 captions in the image
- Switch to the mentioned branch and refresh the editor. You won't see the issue anymore.
- Disable the caption, the bottom spacing issue is due to an extra caption. it also get fixed.

### Checklist:
